client/elements/menus/sc-more-menu.js

```diff
@@ -170,32 +96,45 @@ class SCMoreMenu extends LitLocalized(LitElement) {
       localizedStringsPath: { type: String },
       compactViewChosen: { type: Boolean },
       routeName: { type: String },
+      languageIsVisible: { type: Boolean },
     };
   }
 
   constructor() {
     super();
     this.menuCreated = false;
     this.localizedStringsPath = '/localization/elements/sc-more-menu';
+    this.languageListResponse = [];
     this.routeName = store.getState().currentRoute.name;
     this.alwaysShowUniversalToolbar = store.getState().alwaysShowUniversalToolbar;
+    this.languageIsVisible = store.getState().languageMenuVisibility;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
```

Comment:
can be removed since it doesn't do anything

Reply:
Removed in 07643a2f